### DISCUSSION TITLE
Connection between Flare and Exiled Kingdoms.

### DISCUSSION THREAD

**Discussion by Vecna-max** [flareteam/flare-game#970] [Q&A]
I've noticed a striking resemblance between exiled kingdom (an mobile game it is one IOS and Android I believe) character art, and design elements and flares character art, and design elements. They look almost exactly the same and I'm just wondering if the artist for the characters is connected or is Flare also a passion project from the developers who made exiled kingdoms. There is a character in exiled kingdoms who is a champion called Adaon he has the same character art as the wondering trader you first meet in the goblin camp area by the goblin cave. It is not similar but is exactly the same, just curious if you guy contributed to it. There are some designs in Exiled Kingdoms that also found in flare as well the above ground coffins (/art_src/tilesets/dungeon/tombwoof.blend (sarcophagus mesh)

Here is a link to Exiled Kingdoms Wiki to Adaon page: https://exiledkingdoms.com/wiki/index.php/Adaon

**Comment by dorkster** [comment]
Yes, this comes up often. The assets made for Flare are free to use and remix as long as attribution is given. The developer for Exiled Kingdoms used our assets (as well as some from the modding community) under those terms. Other than that, we don't have any connection to their game.

You can browse the Flare collection on OpenGameArt: https://opengameart.org/content/flare
Most of the assets are by @clintbellanger, and our character portraits were done by Justin Nichol.
